standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -3019,9 +3036,15 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
   bool delete_partition_column_statistics(1:string db_name, 2:string tbl_name, 3:string part_name, 4:string col_name, 5:string engine) throws
               (1:NoSuchObjectException o1, 2:MetaException o2, 3:InvalidObjectException o3,
                4:InvalidInputException o4)
+  bool delete_partition_column_statistics_req(1:DeletePartitionColumnStatisticsRequest req) throws
+              (1:NoSuchObjectException o1, 2:MetaException o2, 3:InvalidObjectException o3,
+              4:InvalidInputException o4)
   bool delete_table_column_statistics(1:string db_name, 2:string tbl_name, 3:string col_name, 4:string engine) throws
               (1:NoSuchObjectException o1, 2:MetaException o2, 3:InvalidObjectException o3,
                4:InvalidInputException o4)
+  bool delete_table_column_statistics_req(1: DeleteTableColumnStatisticsRequest req) throws
```

Comment:
can we merge the two methods? i.e, merge them to `bool delete_column_statistics_req(DeleteColumnStatisticsRequest req)`